tests/CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ if(AVIF_ENABLE_FUZZTEST)
     add_avif_fuzztest(avif_fuzztest_read_image)
     add_avif_fuzztest(avif_fuzztest_yuvrgb)
 
-    if(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    if(AVIF_ENABLE_GAIN_MAP)
```

Comment:
I wonder if we should remove `_experimental` from these two fuzztest names. Note: Simply removing `_experimental` will conflict with other fuzztest names. Need to look into this.

Reply:
The fuzzers can indeed be merged but this can be done in another PR I think.